src/dump.c

```diff
@@ -567,6 +567,11 @@ static void jl_serialize_value_(jl_serializer_state *s, jl_value_t *v, int as_li
         jl_serialize_value(s, ((jl_tvar_t*)v)->lb);
         jl_serialize_value(s, ((jl_tvar_t*)v)->ub);
     }
+    else if (jl_is_vararg_marker(v)) {
+        write_uint8(s->s, TAG_VARARG);
+        jl_serialize_value(s, ((jl_vararg_marker_t*)v)->T);
+        jl_serialize_value(s, ((jl_vararg_marker_t*)v)->N);
+    }
```

Comment:
Did you encounter any particular reason this is needed? I would think the default case would work for these.

Reply:
No, I thought I did, but I think the default is fine. I'll try removing it and see if things work out.